### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by stackpoint**
Please see commits for documentation.

### CODE REVIEW

**Comment by stackpoint** [comment]
@Stiopa866

File: events/ME_Australia_Events.txt [hunk 1]
```diff
@@ -98,7 +98,6 @@ country_event = { # §[Root.GetAdjective]§! Rise to Power
 	desc = ME_Australia_Events.2.D
 	picture = ENLIGHTENMENT_eventPicture
 
-	is_triggered_only = yes
```
Comment (stackpoint) [inline events/ME_Australia_Events.txt:101]: Searched for `ME_Australia_Events.2` and did not find an event call for it in the mod files.

image

File: events/ME_Canada_Events.txt [hunk 2]
```diff
@@ -931,7 +931,6 @@ country_event = {
 	picture = REVOLT_eventPicture
 
 	fire_only_once = yes
-	is_triggered_only = yes
```
Comment (stackpoint) [inline events/ME_Canada_Events.txt:934]: Searched for `ME_Canada_Events.17` and did not find an event call for it in the mod files.

image

**Comment by Stiopa866** [comment]
<3

**Pull request merged**